### DISCUSSION TITLE
Can't install shellcheck

### DISCUSSION THREAD

**Discussion by gr8mustachio** [Homebrew/discussions#3647] [Getting started]
Intel MacBook Pro
when I enter the command 'brew install shellcheck' I get this error:
Warning: No available formula with the name "shellcheck".
==> Searching for similarly named formulae...
Error: No similarly named formulae found.
==> Searching for a previously deleted formula (in the last month)...
Error: No previously deleted formula found.
==> Searching taps on GitHub...
Error: No formulae found in taps.

I can't install cabal either, anyone know how to solve this issue? Thanks

**Comment by p-linnane** [comment]
What is the output of `brew config` and `brew doctor`? Did you run `brew update-reset && brew update`?
